Makefile

```diff
@@ -0,0 +1,13 @@
+dind/% operator/% runner/%:
+	$(MAKE) -C $(@D) $(@F)
+
+upgrade:
+	$(eval LATEST := $(shell curl -s https://api.github.com/repos/actions/runner/releases/latest | jq -r '.tag_name' | sed -E 's/^v//'))
+	exit $(.SHELLSTATUS)
+	sed -Ei "s/^AGENT_VERSION \?= .*/AGENT_VERSION ?= $(LATEST)/" ./operator/Makefile
+	sed -Ei "s/^RUNNER_VERSION \?= .*/RUNNER_VERSION ?= $(LATEST)/" ./runner/Makefile
+	git add -A
+	git commit -m "feat: actions/runner#v$(LATEST)"
+	git checkout -b "feat/actions-runner-v$(LATEST)"
+	git push
+	gh pr create --title "Actions Runner v$(LATEST)"
```

Comment:
> `exit $(.SHELLSTATUS)`

Was that intentional or is it leftover from tests?

We are counting on `git commit` to fail here if we're already up-to-date, right? But that means every minute there will be an failling workflow execution. Wouln't be better to end successfully either way?

Reply:
Regarding the `exit $(.SHELLSTATUS)`, the problem is that when you run a subshell inside a Makefile, if it fails nothing happens. To make it fail if the subshell exits with a non-zero exit code, I run the `exit` command with the subshell exit code.

Reply:
About your second question, yes I was relying on the `git commit` to fail and it would cause the workflow to fail as well. This behavior has been changed in my last commit.

Reply:
I didn't understand how the `exit` command will have its effect only if the subshell exits with non-zero exit code.

Reply:
It is because make forks on every command and it treats exit just like a regular command that exited with 0 or non-0 exit code.